lib/components/SoundMaker/index.jsx

```diff
@@ -173,6 +173,10 @@ export class SoundMaker extends Component {
     this.props.stopAllSounds()
   }
 
+  saveSound = () => {
+    this.props.saveSound(this.state.spec, 1)
+  }
```

Comment:
@dylanavery720  Should the sound be configured to JSON before passing it to the action?